test.sh

```diff
@@ -41,10 +41,12 @@ done
 echo "Main: ./test/Tests.ooc" >> "$TESTS_USE_FILE"
 rm -f .libs/tests-linux64.*
 rock -q +-Wall $ARGS $FLAGS $FEATURES $TESTS_USE_FILE
-if [[ $? -eq 0 && -f ./Tests ]]; then
+exitcode=$?
+if [[ $lol -eq 0 && -f ./Tests ]]; then
 	./Tests
+	exitcode=$?
 fi
-if [[ !( $? == 0 ) ]]
+if [[ !( $exitcode == 0 ) ]]
```

Comment:
I can't really remember to be honest, but if I had to guess, we had reason to either return `1` or `0` to keep it simple.